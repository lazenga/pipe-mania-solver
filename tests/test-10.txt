VC	BE	VD
BB	VE	LH
VE	LV	VB