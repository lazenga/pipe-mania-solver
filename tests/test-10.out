VB	BB	VE
BD	VC	LV
VD	LH	VC
